subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -585,18 +602,44 @@ static void isr_rx(void *param)
 
 	/* Enqueue PDUs to ULL */
 	node_rx = NULL;
+	lll->stream_curr = 0U;
 	payload_index = lll->payload_tail;
 	for (bis_idx = 0U; bis_idx < lll->num_bis; bis_idx++) {
+		struct lll_sync_iso_stream *stream;
 		uint8_t payload_tail;
+		uint8_t str_idx;
```

Comment:
What is `str` in this context? Stream? 

May be risky to use `str` as that is usually the abbreviate for `string`.